distributed/bokeh/scheduler.py

```diff
@@ -328,13 +328,13 @@ def update(self):
                 else:
                     processing_color.append('blue')
 
-            nbytes = [ws.info['memory'] for ws in workers]
+            nbytes = [ws.metrics['memory'] for ws in workers]
```

Comment:
@jhamman you'll probably want to be aware of this change for https://github.com/dask/dask-kubernetes/pull/101